Snippets/Snippets_3/DataBus/CleanupStrategy/Usage.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Snippets3.DataBus.CleanupStrategy
+{
+    using System;
+    using System.IO;
+    using Snippets3.DataBus.DataBusProperty;
+
+    public class Usage
+    {
+        #region FileLocationForDatabusFiles
+        public void Handle(MessageWithLargePayload message)
+        {
+            string filename = Path.Combine(@"c:\databus_files\", message.LargeBlob.Key);
```

Comment:
@dvdstelt - Often times whether we like it or not, users pick stuff from samples. So, I am all for scheduling a delete after x days policy and executing the cleanup as opposed to doing it in a message handler.